### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by saschat**

### CODE REVIEW

File: src/content/docs/docs/features/events.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Handle events
```
Comment (paddyohanlon) [inline src/content/docs/docs/features/events.mdx:2]: Handle **E**vents

**Comment by saschat** [comment]
The thing about login events is that they are integral to authentication so moving them to "handle events" is not an option. Maybe copying them over there?

**Comment by paddyohanlon** [comment]
> The thing about login events is that they are integral to authentication so moving them to "handle events" is not an option. Maybe copying them over there?

I think copying is good.

File: src/content/docs/docs/start-here/local-development.md [hunk 2]
```diff
@@ -2,4 +2,38 @@
...
+
+You can change these defaults with the following options.
+
+```
+npx bzr [options]
+  -i specify app ID. It is optional and defaults to 'test'.
+  -u specify app URL. It is optional and defaults to 'http://localhost:8080'.
+  -r specify redirect URI. Can be specified multiple times. It is optional and defaults to 'http://localhost:8080'.
+  -p specify port. Default is 3377.
+  -h show this help
```
Comment (paddyohanlon) [inline src/content/docs/docs/start-here/local-development.md:22]: A sample command would be nice.

**Comment by paddyohanlon** [comment]
@saschat What do you think about putting the login events in a component to share in the two places?

**Comment by saschat** [comment]
Not sure. The login event sections are not exactly the same, so I am not convinced it is that helpful.

**Comment by paddyohanlon** [comment]
Ok, I assumed they were the same.

**Pull request merged**